test/modules/standard/math/promotion/gen/make_promotion_tests.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# Create promotion tests
+def write_test(name, min, max):
+    filename = '../' + name + '.chpl';
```

Comment:
To make this script work from directories other than test/modules/standard/math/promotion/gen/, you could use something like:

```python
```